src/Solutions/Folds.idr

```diff
@@ -13,16 +13,12 @@ import Data.Vect
 -- 1
 
 anyList : (a -> Bool) -> List a -> Bool
-anyList p []        = False
-anyList p (x :: xs) = case p x of
-  False => anyList p xs
-  True  => True
+anyList p Nil       = False
+anyList p (x :: xs) = p x || anyList p xs
 
 allList : (a -> Bool) -> List a -> Bool
-allList p []        = True
-allList p (x :: xs) = case p x of
-  True  => allList p xs
-  False => False
+allList p Nil       = True
```

Comment:
Please do not delete the previous version but give your version a new name.

Reply:
done: 2b77f31d55888521fb0995730ddd7482715f4e36